js/modules/loki_rpc.js

```diff
@@ -67,20 +67,42 @@ const sendToProxy = async (options = {}, targetNode) => {
   const response = await nodeFetch(url, firstHopOptions);
   process.env.NODE_TLS_REJECT_UNAUTHORIZED = 1;
 
-  const ciphertext = await response.text();
-
-  const ciphertextBuffer = dcodeIO.ByteBuffer.wrap(
-    ciphertext,
-    'base64'
-  ).toArrayBuffer();
+  // FIXME: handle nodeFetch errors/exceptions...
 
-  const plaintextBuffer = await window.libloki.crypto.DHDecrypt(
-    symmetricKey,
-    ciphertextBuffer
-  );
+  const ciphertext = await response.text();
+  let plaintext
+  try {
+    // removed the following part to match the check we have in loki_app_dot_net_api.js
+    // ; not done syncing;
+    if (ciphertext.match(/Service node is not ready: not in any swarm/)) {
```

Comment:
Could we potentially check for a http code instead of matching exact text?

Reply:
Looks like it so far